### ISSUE TITLE
Get docker instructions into readme.md

### ISSUE THREAD

**Issue by TRnonodename** [thomsonreuters/CM-Well#58]
I can only stare at 'coming soon' for so long :-)

**Comment by e-orz** [comment]
@hochgi It's not just documentation. The whole docker image should be reconstructed. Also, automated checks for the docker images should be implementated.

**Comment by TRnonodename** [comment]
Good point...this is just the painful reminder on the front page :-) Do we want a wider 'do docker' issue and add this as an acceptance criteria?

**Comment by e-orz** [comment]
Yes, thanks. It sounds much better.

**Comment by tibotiber** [comment]
Hi! Any update on this?
